make_img.sh

```diff
@@ -4,14 +4,15 @@
 # 256MB bootable FAT32L partition with official Alpine linux and lncm-box
 # Make sure "parted", "dosfstools" and "zip" are installed
 
-OUTPUT_VERSION=v0.5.2 # For Outputing an image
-REL=v3.10 # Which alpine release directory
+OUTPUT_VERSION=v0.5.3 # For Outputing an image
+REL=v3.11 # Which alpine release directory (Grab from https://www.alpinelinux.org/downloads/)
```

Comment:
Perhaps we should use a full version here, as even patch release can REKT things for us.

Reply:
REL is used in the directory
OUTPUT_VERSION is used in the file

That confused me as well when I saw it